install_local.sh

```diff
@@ -19,7 +19,10 @@ if ! command -v gdown &> /dev/null; then
     exit 1
 fi
 
-CRT_DIR=$(pwd)
+CRT_DIR=$(pwd)                                 # .../intball2_ws/src/int-ball2_isaac_sim
+SRC_DIR=$(dirname "$CRT_DIR")                  # .../intball2_ws/src
+WS_DIR=$(realpath "$SRC_DIR/..")               # .../intball2_ws
+
 cd int-ball2_isaac_sim
```

Comment:
Since we are using the above, we could change this to:
```suggestion
cd $PKG_DIR
```